spec/features/admin_csv_download_spec.rb

```diff
@@ -20,6 +20,19 @@
       csv = CSV.parse(body)
       expect(csv.count).to eq(submitted_claims.count + 1)
     end
+
+    scenario "User downloads a CSV for the payroll provider" do
+      submitted_claims = create_list(:claim, 5, :submittable, submitted_at: DateTime.now)
+      create_list(:claim, 2, :submittable)
+
+      expect(page).to have_link(nil, href: payroll_admin_claims_path(format: :csv))
+      click_on "Download payroll data"
+
+      expect(page.response_headers["Content-Type"]).to eq("text/csv")
+
+      csv = CSV.parse(body)
+      expect(csv.count).to eq(submitted_claims.count + 1)
```

Comment:
can we add some more specific expectations here that will give us confidence the file being returned is definitely the payroll one?